lib/api/CJsonOutputWriter.cc

```diff
@@ -909,6 +913,15 @@ void CJsonOutputWriter::writeCategoryDefinition(int categoryId,
         m_Writer.String(example);
     }
     m_Writer.EndArray();
+    m_Writer.String(NUM_MATCHES);
+    m_Writer.Uint64(numMatches);
+    m_Writer.String(PREFERRED_TO_CATEGORIES);
+    m_Writer.StartArray();
+    for (TIntVecCIter iter = usurpedCategories.begin(); iter != usurpedCategories.end(); ++iter) {
+        int id = *iter;
+        m_Writer.Int(id);
+    }
```

Comment:
```suggestion
    for (int id : usurpedCategories) {
        m_Writer.Int(id);
    }
```

(There might be other places where we still have the verbose form because for many years the code had to work with old compilers.)